src/components/WeekNumber.js

```diff
@@ -4,9 +4,11 @@ import { connect } from 'react-redux';
 
 const WeekNumber = ({ week }) => (
 
-  <div className="WeekNumber widget">
-    <div className="WeekNumber-title widget-title">Week</div>
-    <div className="WeekNumber-number widget-number">{week}</div>
+  <div className="panel panel-primary">
```

Comment:
for example instead of using panel, with Material UI you need a http://www.material-ui.com/#/components/card